### ISSUE TITLE
AJAX Call Failed!

### ISSUE THREAD

**Issue by Nairdaa** [sbpp/sourcebans-pp#748]
What are the steps to reproduce this issue?
-------------------------------------------
1. Click the Login button
2. Enter username
3. Enter password
4. Click "Login"

What happens?
-------------
![image](https://user-images.githubusercontent.com/38843773/130527047-777a5a8c-3cc0-4649-91c5-8c84e6646f53.png)

What were you expecting to happen?
----------------------------------
To log in.

Any logs, error output, etc.?
----------------------------

As in the image above.

Any other comments?
-------------------
Downgrading the PHP version is a no go.

What versions of software are you using?
----------------------------------------
**Operating System:** Ubuntu 18.04
**SourceBans++ Version:** Newest master branch, there were even more errors on 1.8.3 release and 1.8.2.
**PHP Version:** PHP 8.0.5 (cli) (built: May  3 2021 11:30:38) ( NTS )
**MySQL Version:** mysql  Ver 15.1 Distrib 10.3.29-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2
**Link to your project:** http://148.251.79.234:82/
**Link to a phpinfo() output:** http://148.251.79.234/phpinfo.php

**Comment by Nairdaa** [comment]
SB_NEW_SALT got solved.

Error: the XML response that was returned from the server is invalid. Received:
Warning: Undefined array key "HTTPS" in /usr/share/sourcebans/includes/auth/Auth.php on line 37


Here is my error now when I try to log in with correct credentials. If I put wrong ones I get regular message that user/pass were incorrect.
If I put the correct credentials that's the error I get and I never get logged in, like the session was f**ked up. I tried removing cache files etc.
